packages/manager/modules/bm-server-components/src/general-information/general-information.controller.js

```diff
@@ -15,9 +15,7 @@ export default class BmServerComponentsGeneralInformationController {
     const { os } = this.server;
     if (!os || Object.values(OPERATING_SYSTEM_ENUM).includes(os)) {
       return this.$translate.instant(
-        os === OPERATING_SYSTEM_ENUM.BRING_YOUR_OWN_IMAGE
-          ? 'dedicated_server_dashboard_distribution_byoi'
-          : 'dedicated_server_dashboard_distribution_none',
+        'dedicated_server_dashboard_distribution_none',
```

Comment:
For those who already have a BYOI, could this cause translation problems?

Reply:
Good point: I'm going to replace in the backend all servers with OS byoi to byoi_64, so last OS will be displayed as Bring Your Own Image rather than None. So we avoid keeping legacy stuffs. I'm going to declare the CHANGE for the following week (customers will not be impacted), so it will be ready when this PR will be deployed.